### DISCUSSION TITLE
Server not responding: HttpErrorResponse: Http failure response for http://localhost:8082/getPageList: 0 Unknown Error

### DISCUSSION THREAD

**Discussion by Pawanupadhyay10** [polypheny/Polypheny-DB#323] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52281814/113688432-ea26fc00-96e6-11eb-8a7b-86cc7a7f5228.png)

after setup of polypheny-UI 

why this error is comming
"Server not responding:
HttpErrorResponse: Http failure response for http://localhost:8082/getPageList: 0 Unknown Error "

**Comment by datomo** [comment, marked as answer]
Hey @Pawanupadhyay10 
It looks like you don't have Polypheny-DB running, only Polypheny-UI.
I guess you started Polypheny-UI by executing `ng serve`?

For this to work correctly you need to clone Polypheny-DB and start it first.
The easiest way to do this would be to open Polypheny-DB in Intellij or Eclipse and run the PolyphenyDB class in the dbms module.
After that, you will be able to access Polypheny directly via localhost:8080.

To develop for Polypheny-UI you additionally can start the development mode of the UI by running `ng serve` in Polypheny-UI.
After that you can access the development instance of the UI via localhost:4200.

I hope this helps, if not feel free to comment on this post with additional questions regarding the problem.

Please refrain from creating additional post for the similar question, as it tends to clutter the disscussion board and makes it harder to keep track.

**Reply by Pawanupadhyay10** [reply]
hello  datmo, i have imported polypheny-db and build it but i am not getting  "run the PolyphenyDB class in the dbms module"
Can ,you pls reccommend me some youtube  vedios or blogs. Or pls tell me how to do it because  i am new to intelij.
![image](https://user-images.githubusercontent.com/52281814/113702743-8ce77680-96f7-11eb-8535-847e4ab4d89e.png)

**Reply by datomo** [reply]
When you open the dbms folder -> src -> main -> java -> org.polypheny.db, you find the class PolyphenyDB.
By rightclicking this class you should be able to "run" it.

![grafik](https://user-images.githubusercontent.com/5930207/113711554-5ef93680-96e5-11eb-87ef-671af4caa65e.png)

If you don't see this option you probably need to configure the Java SDK for your Intellij first.

For this you would need to navigate to **File** -> **Project Structure** and select a appropriate Java SDK(8 up to 15 should work) on you system.

I hope this helps.

**Reply by Pawanupadhyay10** [reply]
thanks ,finally it completed.
